### DISCUSSION TITLE
error: (-215:Assertion failed) borderType!= BORDER_CONSTANT in function 'pyrDown'

### DISCUSSION THREAD

**Discussion by msmartin4470** [rainyl/opencv_dart#50] [Q&A]
Could you help me out with this error message?

CvException (CvException(code:
ErrorCode (-215), err:, msg:OpenCV (4.9.0) / Users/runner/work/opencv.full/opencv.full/build/ opencv/modules/imgproc/src/
pyramids.cpp:1415: error: (-215:Assertion failed) borderType!= BORDER_CONSTANT in function
'pyrDown'
) in pyrDown of file /Users/runner/work/ opencv.full/opencv.full/build/opencv/modules/ imgproc/src/pyramids.cpp:1415)

Te border type is a default parameter that I am not changing, so that is why I am a little confused with the issue.  Thanks.

**Comment by rainyl** [comment]
The default bordertype in dart side is wrong, I will fix it soon, for now you can explicitly set the borderType to make it work.

**Reply by rainyl** [reply]
@msmartin4470 fixed in v1.0.3+2
